mingw-w64-arrow/PKGBUILD

```
@@ -33,7 +33,7 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
              "${MINGW_PACKAGE_PREFIX}-meson")
 options=("staticlibs" "strip" "!buildflags")
 source=(https://github.com/apache/arrow/archive/apache-arrow-${pkgver}.tar.gz)
-sha256sums=('4db2233c25d1ef14f90f9de8e9d808a2d386c67e7116405ddd22d8f981fe66c1')
+sha256sums=('cbc51c343bca08b10f7f1b2ef15cb15057c30e5e9017cfcee18337b7e2da9ea2')
```

```suggestion
sha256sums=('ecb6da20f9288c0ca31f9b457ffdd460198765a8af27c1cac4b1382a8d130f86')
```

Because this package uses https://github.com/apache/arrow/archive/apache-arrow-0.17.1.tar.gz not https://downloads.apache.org/arrow/arrow-0.17.1/apache-arrow-0.17.1.tar.gz .

---

Just looked up for it :)